resources/schemas/dbscripts/postgresql/targetedms-25.000-25.001.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE targetedms.QCMetricConfiguration RENAME COLUMN TimeValue To MinTimeValue;
+ALTER TABLE targetedms.QCMetricConfiguration ADD COLUMN MaxTimeValue REAL;
```

Comment:
We need to set values to avoid breaking existing configs.

I think that means something like:

UPDATE targetedms.QCMetricConfiguration SET MaxTimeValue = 1000 WHERE MinTimeValue IS NOT NULL;
UPDATE targetedms.QCMetricConfiguration SET TimeValueOption = 'First' WHERE MinTimeValue IS NOT NULL;

Can you give this a test and commit?